storybookjs/storybook
# Build: Add installScripts step in bootstrap command

Issue: N/A

## What I did

A few things:
- The dependencies from the `scripts` folder are now updated as a prerequisite of the `install` step from the bootstrap command. This will happen when running `yarn bootstrap --core` or `yarn bootstrap --install`
- The bootstrap command now gracefully handles quitting with CTRL+C
- Some scripts were updated as a result of this change
- Small documentation was updated 

## How to test

You can remove `node_modules` from `code` and `scripts` then run:
```
./bootstrap --core
```

and everything should be good to go.


If your answer is yes to any of these, please make sure to include it in your PR.
